docs/system_overview/configuration.md

```diff
@@ -367,7 +367,7 @@ This file defines the behavior of the server Agent that answers individual reque
     // This must be the same across all iRODS servers in a Zone.
     "server_port_range_end": 20199,
 
-    // The authentication scheme used by the zone_user: native, PAM, KRB, or GSI.
+    // The authentication scheme used by the zone_user: native, pam_password, KRB, or GSI.
```

Comment:
Hmm... actually, KRB and GSI are not supported anymore (for now). We should probably remove those two from the list.

Reply:
No problem!  Its updated.